noinst/specs/update_specs.R

```diff
@@ -22,10 +22,10 @@ SPECS <- read_csv("noinst/specs/SPECS.csv") |>
   as.data.frame()
 
 
-save(SPECS, file = "data/specs.RData", version = 2)  # version 3 requires >= R3.5
 
 
 
+usethis::use_data(SPECS, overwrite = TRUE)
```

Comment:
Does this keep `version = 2`?

Reply:
Yes, `version = 2` is the default in `usethis::use_data`.